src/sidebar/views/overview.ts

```diff
@@ -16,91 +16,87 @@ export class OverviewItem {
+            new OverviewItem('or set the output folder to get started')
+        ],
+        'normal': [
+            new OverviewItem(() => `Total files analyzed: ${ExtensionApi.metadata.sourceFiles.size}`),
+            new OverviewItem(() => `Last analysis run: ${
+                new Date(ExtensionApi.metadata.metadata!.timestamps.begin /* seconds */ * 1000).toLocaleString()
+            }`),
+            new OverviewItem(() => {
+                // Time in seconds
+                const beginTime = ExtensionApi.metadata.metadata!.timestamps.begin;
+                const endTime = ExtensionApi.metadata.metadata!.timestamps.end;
+                const interval = endTime - beginTime;
+
+                const hours = Math.floor(interval/3600) % 60;
+                const minutes = Math.floor(interval/60) % 60;
+                const seconds = Math.floor(interval) % 60;
+                const ms = Math.floor(interval * 1000) % 1000;
+
+                if (hours > 0 || minutes > 0) {
+                    // H:MM:SS s / M:SS s
+                    const formattedMinutes = hours > 0 ? (`${hours}:${minutes.toString().padStart(2, '0')}`) : minutes;
+                    return `Analysis duration: ${formattedMinutes}:${seconds.toString().padStart(2, '0')} s`;
+                } else {
+                    // S.fff s
+                    return `Analysis duration: ${(seconds + ms / 1000).toFixed(3)} s`;
+                }
+            }),
+            new OverviewItem(() => `Used analyzers: ${
+                Object.keys(ExtensionApi.metadata.metadata!.analyzers).join(', ')
+            }`),
+        ]
     };
 
-    private regularItemsList = [
-        'files',
-        'lastRun',
-        'buildLength',
-        'analyzers',
-
-        'separator',
-
-        'reloadMetadata',
-        'analyzeCurrentFile',
-        'analyzeProject',
-    ];
-    private notFoundItemsList = [
-        'notfound',
-        'notfound2',
-        'notfound3',
+    private bottomItems: {[id: string]: OverviewItem[]} = {
+        'normal': [
+            new OverviewItem('Reload CodeChecker metadata', {
+                title: 'reloadMetadata',
+                command: 'codechecker.backend.reloadMetadata',
+            }),
+            new OverviewItem('Re-analyze current file', {
+                title: 'reloadMetadata',
+                command: 'codechecker.executor.analyzeCurrentFile',
+            }),
+            new OverviewItem('Re-analyze entire project', {
+                title: 'reloadMetadata',
+                command: 'codechecker.executor.analyzeProject',
+            }),
+        ],
+        'ccNotFound': [
+            new OverviewItem('Reload CodeChecker metadata', {
+                title: 'reloadMetadata',
+                command: 'codechecker.backend.reloadMetadata',
+            }),
+            new OverviewItem('——'),
+            new OverviewItem('compile_commands.json not found.'),
```

Comment:
Why do we need this item? I created the compilation database but this item is still there and it is missleading.

Reply:
It should automatically hide itself when the user selects a database.
If the sidepanel doesn't reload when the database is created, that's a bug.

I should add watches to each of the database paths, and fire an event when one's changed.

Reply:
Added watches for the files, so the sidebar should update when `CodeChecker log` finished running.